content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -280,6 +280,27 @@ ensure that no two Services can collide. Kubernetes does that by allocating each
 Service its own IP address from within the `service-cluster-ip-range`
 CIDR range that is configured for the {{< glossary_tooltip term_id="kube-apiserver" text="API Server" >}}.
 
+#### Static subrange nodeport
+
+A static subrange is a subset of the default NodePort range that is 
+reserved for a specific service. You can define a static subrange by 
+specifying a range of port numbers within the default range that are 
+reserved for your service. For example, you might reserve ports 
+31000-31009 for your service.
```

Comment:
The [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3668-reserved-service-nodeport-range) talks about dynamic and static subranges. Do we need to document both kinds?